Modules/IO/NIFTI/test/itkNiftiImageIOTest12.cxx

```diff
@@ -25,33 +25,42 @@ itkNiftiImageIOTest12(int ac, char * av[])
 {
   //
   // first argument is passing in the writable directory to do all testing
-  if (ac > 1)
+  if (ac != 3)
   {
-    char * testdir = *++av;
-    --ac;
-    itksys::SystemTools::ChangeDirectory(testdir);
-  }
-  if (ac != 2)
+    std::cerr << "Incorrect command line usage:" << std::endl;
+    std::cerr << av[0] << " <TempOutputDirectory> <filename>" << std::endl;
```

Comment:
av[0] -> itkNameOfTestExecutableMacro(av)
from https://github.com/InsightSoftwareConsortium/ITK/blob/091d0f74209a0e1a032e4cb16f5b6b4bab18045e/Modules/Core/TestKernel/include/itkTestingMacros.h#L260-L264

Reply:
Fixed.  Thank you.